src/index.css

```diff
@@ -1,27 +1,79 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Open+Sans:ital,wght@0,500;1,500&display=swap');
+
+:root {
+  --color-display: #858393;
+  --color-secondary: #e0e0e0;
+  --color-tertiary: #f4913e;
+  --color-white: #fff;
+  --transition: all 0.3s ease-in;
+}
+
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+
+  /* font-size: 62.5%; */
```

Comment:
- [ ] Excellent job with the requirement so far but I think as one of the principles of best practices, not commenting unused codes is a great virtue that could save one from messy code or being prone to bugs. Kindly remove this line to tidy up your codespace.